web/components/contract/contract-tabs.tsx

```diff
@@ -18,14 +27,18 @@ export function ContractTabs(props: {
   comments: ContractComment[]
   tips: CommentTipMap
 }) {
-  const { contract, user, bets, tips } = props
+  const { contract, user, tips } = props
   const { outcomeType } = contract
 
+  const updatedBets = useBets(contract.id, {
+    filterChallenges: false,
+    filterRedemptions: true,
```

Comment:
Redemptions are needed within the `userBets` to compute accurate invested/profit numbers, even if they don't appear in the list. Redemptions are not needed for the public bets list.

Reply:
OK, I'll make sure everything like this is getting computed correctly.